apps/emqx_authn/src/emqx_authn_password_hashing.erl

```diff
@@ -51,6 +52,7 @@
          hash/2,
          check_password/4]).
 
+namespace() -> "authn-hash".
```

Comment:
For my own information: is this to set up the URL path in the docs?

Reply:
yes, this only affects documentation.